### ISSUE TITLE
Other issue involving the wiitar on wiitarthing

### ISSUE THREAD

**Issue by nin52** [InvoxiPlayGames/FestivalInstrumentMapper#38]
Heres a short video i made showing the issue

https://youtu.be/0q1mJi0LMqI

**Comment by InvoxiPlayGames** [comment]
Have you followed the steps in "Set up Xbox 360 Controller Hiding"?

**Comment by nin52** [comment]
> Have you followed the steps in "Set up Xbox 360 Controller Hiding"?

Yes I did, everything is enabled and I am still getting double inputs
